docs/src/catalyst_applications/bifurcation_diagrams.md

```diff
@@ -1,115 +1,118 @@
+
+
+## Systems with conservation laws
+Some systems are under-determined, and for a given parameter set they have an infinite number of possible steady states, preventing bifurcation diagrams from being computed. Similar to when we [compute single steady states](@ref homotopy_continuation_conservation_laws), we can utilise Catalyst's ability to detect and eliminate conservation laws to resolve this issue. This requires us to provide information of the species concentrations at which we wish to compute the bifurcation diagram. These are provided to the `BifurcationProblem` using the `u0` argument.
+
+To illustrate this, we will create a simple model of a kinase that is produced and degraded (at rates *p* and *d*). The kinase facilitates the phosphorylation of a protein (*X*), which is dephosphorylated at a constant rate. For this system, we will compute a bifurcation diagram, showing how the concentration of the phosphorylated protein (*Xp*) depends on the degradation rate of the kinase (*d*). We will set the total amount of protein (*X+Xp*) to *1.0*.
+```@example ex2
+using BifurcationKit, Catalyst, Plots
+kinase_model = @reaction_network begin
+    (p, d), 0 <--> K
+    (K*kP,kD), X <--> Xp
+end
+
+u_guess = [:K => 1.0, :X => 1.0, :Xp => 1.0]
+p_start = [:p => 1.0, :d => 0.5, :kP => 2.0, :kD => 5.0]
+u0 = [:X => 1.0, :Xp => 0.0]
+bprob = BifurcationProblem(kinase_model, u_guess, p_start, :d; plot_var=:Xp, u0=u0)
+
+p_span = (0.1, 10.0)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], max_steps = 1000)
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+plot(bif_dia; xguide="d", yguide="Xp")
 ```
+This bifurcation diagram does not contain any interesting features (such as bifurcation points), and only shows how the steady state concentration of *Xp* is reduced as *d* increases. For this example, we will note two facts:
+- When providing the concentrations for computing the conserved quantities (in `u0`), we only have to designate the concentrations of species that are actually involved in conservation laws. For larger systems, determining which one are may, however, be difficult. In this case, it might still be wise to provide concentrations for all species.
+- The steady state guess in `u_guess` does not actually have to fulfil the conserved concentrations provided in `u0`.
```

Comment:
Is this really true? Can you add an explanation of the difference between `u_guess` and `u0` here and how they are both used?

Reply:
`u_guess` is just a guess of the initial steady state.

`u0` is used to compute conservation laws (e.g. what the conserved quantity X1+X2 should be). Yes, it is annoying to have to have both, but feel like if we just use `u_guess` people will do something trivial (like `ones`), and never realise that it is used for conserved quantities computation.

Reply:
rewritten this to make everything extra clear